apps/checker/app/services/EntityHelper.scala

```diff
@@ -26,6 +32,45 @@ class EntityHelper() {
 
   val allFinders: List[NameFinderME] = List(personFinder, organizationFinder, locationFinder)
 
+  case class NERResult(label: String, text: String, start: Int, end: Int)
+
+  object NERResult {
+    implicit val reads: Reads[NERResult] = Json.reads[NERResult]
+  }
+
+  def getEntityResultFromNERService(text: String): Future[Either[Error, List[String]]] = {
+    val url = config.nerApiUrl
+    val key = config.nerApiKey
+    val model = "en_core_web_trf"
+    val entityTypes = List("PERSON", "NORP", "FAC", "LOC", "GPE", "PRODUCT", "EVENT", "WORK_OF_ART")
```

Comment:
Perhaps we could include "ORG", too? Corporation/org names are often novel.

```suggestion
    val entityTypes = List("PERSON", "NORP", "FAC", "LOC", "GPE", "PRODUCT", "EVENT", "WORK_OF_ART", "ORG")
```

e.g. 

<img width="680" alt="Screenshot 2023-10-26 at 10 16 54" src="https://github.com/guardian/typerighter/assets/7767575/c6730f6f-82eb-445f-abb9-99a991eea8f0">

but 

```json
{
    "result": [
        {
            "text": "...",
            "ents": [
                {
                    "label": "ORG",
                    "text": "gal-dem",
                    "start": 0,
                    "end": 7
                }
            ]
        }
    ]
}
```

Reply:
Good catch, I've added it